src/geoserver/util.py

```diff
@@ -570,12 +570,26 @@ def delete_wms_layer(geoserver_workspace, layer, auth):
         response.raise_for_status()
 
 
-def patch_wms_layer(geoserver_workspace, layer, *, auth, bbox):
+def patch_wms_layer(geoserver_workspace, layer, *, auth, bbox=None, title=None, description=None):
     wms_layer = get_wms_layer(geoserver_workspace, layer, auth=auth)
     if bbox:
         wms_layer['nativeBoundingBox'] = bbox_to_native_bbox(bbox)
         wms_layer['nativeCRS'] = 'EPSG:3857'
         # automatically recalculates also 'latLonBoundingBox'
+    if title:
+        wms_layer['title'] = title
+        keywords = [
+            "features",
+            layer,
+            title
+        ]
+        keywords = list(set(keywords))
+        wms_layer['keywords'] = {
+            "string": keywords
+        }
+    if description:
```

Comment:
`if description` vs. `if description is not None` - not sure which is more suitable